### DISCUSSION TITLE
How are we storing and communicating information about the duration of an alert?

### DISCUSSION THREAD

**Discussion by hannahker** [OCHA-DAP/hdx-signals#66] [General]
In following [this logic](https://github.com/OCHA-DAP/hdx-signals/blob/5dc35323fe3d90c6e0c0c5b6434173b1ee3fbf28/src/alerts/filter_alerts.R#L11-L19), I'm understanding that an end user is only going to get an alert for a unique country/indicator/alert level if they haven't already gotten one within the past 180 days. Despite only getting one email, this "alert state" could be ongoing. How does an end user know what's happening after they get this initial alert? It looks like this logic will send a subsequent alert if a situation escalates, but what about otherwise? 

Is it correct that someone should interpret a lack of follow up email to be that either 1) the alert is no longer present, or 2) the alert is ongoing at the same level? This seems a bit ambiguous, so I'm wondering if there's a way we should treat these two situations differently (or potentially, there's existing logic that I'm missing). 

cc @zackarno @caldwellst

**Comment by caldwellst** [comment]
The original system was designed to have an alert ongoing or not. However, we've moved away because it's very difficult to communicate and understand, and can be very difficult to determine when an alert should "end". Thus, we should think about Signals as points in time, not spans.

So, not receiving an email is simply that there is no alert. And in the communications, make it clear that alerts are generated when there is a deterioration in some dataset that has not seen similar deteriorations in the recent past.

Does that make sense?

**Reply by hannahker** [reply]
Yes, that makes lots of sense! Definitely cleaner this way 😄 I was having a hard time wrapping my head around how this would have worked.
